src/coreclr/jit/emitxarch.cpp

```diff
@@ -5854,19 +5854,19 @@ bool emitter::IsMovInstruction(instruction ins)
             return true;
         }
 
-#if defined(TARGET_AMD64)
         case INS_movdqa32:
         case INS_movdqa64:
         case INS_movdqu8:
         case INS_movdqu16:
         case INS_movdqu32:
         case INS_movdqu64:
```

Comment:
Couldn't these just be part of the previous case group, that way the ifdef doesn't need to change at all?

Presumably they'd then be ordered directly after `movdqa` and `movdqu`, respectively.

Reply:
Done